src/react_native/biometrics.cljs

```diff
@@ -0,0 +1,76 @@
+(ns react-native.biometrics
+  (:require
+    ["react-native-biometrics" :default biometrics]
+    [clojure.string :as string]
+    [oops.core :as oops]
+    [react-native.platform :as platform]
+    [schema.core :as schema]))
+
+(def ^:const biometric-error-not-available "biometric-not-available")
+(def ^:const biometric-error-not-enrolled "biometric-not-enrolled")
```

Comment:
Biometrics error messages had to be moved here from `constants`, as we're using them in `convert-auth-error-message`.